.gitignore

```diff
@@ -0,0 +1,71 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
```

Comment:
I think anything that is generated by your IDE or Code editor should be ignored globally on your local machine.. It's not good practice for us to be committing those, otherwise we will end up having a huge gitignore file. It only makes sense to ignore stuff that is generated by the project itself that you wouldn't want to commit.